docs/introduction.md

```diff
@@ -0,0 +1,232 @@
+# Introduction
+
+This document is aimed at introducing developers to the Flow Data Provisioning Service project.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+3. [What is Flow](#flow)
+    1. [Glossary](#glossary)
+4. [Developer Guide](#developer-guide)
+    1. [Installation](#installation)
+    2. [Setting up a test environment](#setting-up-a-test-environment)
+5. [More resources](#more-resources)
+
+## Getting Started
+
+The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow [execution state](#execution-nodes).
+
+The reason for this need is that in the execution state, blocks are pruned after a hundred other blocks have been finalized, which makes it impossible to visualize the history of the chain.
+By providing an API which exposes the Flow chain history and implementing the widely used [Rosetta API specification](https://www.rosetta-api.org/), it becomes possible to integrate the Flow blockchain into many 3rd party applications and tools.
```

Comment:
This doesn't really make too much sense. Full state & event history is required to comply with the Rosetta API specification, but it is not one of the main motivations behind the DPS implementation.

The second motivation is that script execution is currently proxied from the access nodes to execution nodes, which is not scalable. DPS is about making the access to the execution state *available* (at any block height) and *scalable* (so it doesn't put load on the network nodes).